src/views/LostItems/LostItemsListContainer/components/LostItemsList/components/RenderActions/components/PatronDetailsLink.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { get } from 'lodash';
+
+import { Button } from '@folio/stripes/components';
+
+import {
+  ACTUAL_COST_RECORD_FIELD_NAME,
+  ACTUAL_COST_RECORD_FIELD_PATH,
+  DEFAULT_VALUE,
+} from '../../../../../../constants';
+
+const PatronDetailsLink = ({ actualCostRecord }) => {
+  const userId = get(actualCostRecord, ACTUAL_COST_RECORD_FIELD_PATH[ACTUAL_COST_RECORD_FIELD_NAME.USER_ID], DEFAULT_VALUE);
+  const isPatronDetailsLinkActive = userId;
```

Comment:
Does it make sense to have an additional variable?

Reply:
For code readability